packages/network/test/integration/UnixSocketWsServer.test.ts

```diff
@@ -0,0 +1,45 @@
+import { ServerWsEndpoint, startHttpServer } from "../../src/connection/ws/ServerWsEndpoint"
+import { PeerInfo } from "../../src/connection/PeerInfo"
+import WebSocket from "ws"
+import { waitForCondition } from "streamr-test-utils"
+
+// eslint-disable-next-line no-underscore-dangle
+declare let _streamr_electron_test: any
+
+describe('ServerWsEndpoint', () => {
+    let serverWsEndpoint: ServerWsEndpoint | undefined = undefined
+
+    afterEach(async () => {
+        try {
+            await serverWsEndpoint?.stop()
+        } catch (err) {
+        }
+    })
+
+    test('WS connection can established with unix sockets', async () => {
```

Comment:
Should probably test these two socket cleanup cases:

socket file exists but nothing listening
socket file exists but another server listening